src/App.js

```diff
@@ -207,7 +207,7 @@ class App extends Component {
       runWalletBackupStatusChecks();
       if (ios) {
         InteractionManager.runAfterInteractions(() => {
-          setTimeout(() => runFeatureUnlockChecks(), 2000);
+          setTimeout(() => runFeatureAndCampaignChecks(), 2000);
```

Comment:
if i'm not mistaken, this was done to run feature unlocks for ios only but for promo sheets we need it for android and ios, we're probably missing android here?

Reply:
mm nice catch